src/deltaE/index.js

```diff
@@ -1,6 +1,17 @@
-export {default as deltaE76} from "./deltaE76.js";
-export {default as deltaECMC} from "./deltaECMC.js";
-export {default as deltaE2000} from "./deltaE2000.js";
-export {default as deltaEJz} from "./deltaEJz.js";
-export {default as deltaEITP} from "./deltaEITP.js";
-export {default as deltaEOK} from "./deltaEOK.js";
+import deltaE76 from "./deltaE76.js";
+import deltaECMC from "./deltaECMC.js";
+import deltaE2000 from "./deltaE2000.js";
+import deltaEJz from "./deltaEJz.js";
+import deltaEITP from "./deltaEITP.js";
+import deltaEOK from "./deltaEOK.js";
+
+export { deltaE76, deltaECMC, deltaE2000, deltaEJz, deltaEITP, deltaEOK };
```

Comment:
```suggestion
// We want both a default export so it can be extended
// as well as individual named exports
export {
	deltaE76, 
	deltaECMC, 
	deltaE2000, 
	deltaEJz, 
	deltaEITP, 
	deltaEOK
};
```